requirements.txt

```diff
@@ -2,7 +2,9 @@ aiohttp>2.0,<4.0
 Pillow
 pystray
 selenium-wire
+yarl
 # use a fork that has an extra option of hiding the unneeded Chrome's console cmd window
 git+https://github.com/sebdelsol/undetected-chromedriver
 # this is installed only on windows
 pywin32; sys_platform == "win32"
+windows-curses; sys_platform == "win32"
```

Comment:
`yarl` is already installed as an `aiohttp` dependency, so I'm not sure why you're adding it there again. `windows-curses` implies this CLI would work only on Windows, while the rest of the application already has Linux support for almost all of it's features, so this entire PR would need to support Linux as well. Not sure why you're using a 3rd party lib when Python 3 comes with one named the same, that already supports both Windows and Linux and appears to have a similar interface: https://docs.python.org/3/library/curses.html

Reply:
`yarl`: Well, strange. PyCharm told me to do so :D I'll remove it.

`windows-curses`: The `curses` lib integrated in Python 3 doesn't fully support Windows (at least doesn't work when I'm testing on my Windows) so `windows-curses` is needed. But on Linux, it works well without additional lib required.